src/components/academy/grading/GradingWorkspace.tsx

```diff
@@ -270,6 +271,15 @@ class GradingWorkspace extends React.Component<GradingWorkspaceProps> {
         icon: IconNames.NINJA,
         body: <Markdown content={props.grading![questionId].question.content} />
       },
+      {
+        label: `Chat`,
+        icon: IconNames.CHAT,
+        body: USE_CHATKIT ? (
+          <ChatApp roomId={props.grading![questionId].grade.comment} />
```

Comment:
I think we should rename the `comment` field to something more meaningful. 

I recommended deprecating the comment field on the backend as well, because you might have unintended behaviors not accounted for. In this case, the action `SUBMIT_GRADING` is submitting an undefined `comment` to the backend, causing the backend to store "undefined" into the comment field on the database.

Reply:
Yea we leave the field as comment for now and see how the chat goes for this summer module.
Will change it afterwards.

Reply:
With respect to the deprecation of comment, Martin would like to have this tested on real users during the summer workshop (14 July). Hence, in the interest of time, we will not be deprecating comment in this pull request as that would require changes on both front and back end.

Reply:
So if I understand this correctly: Every time a TA grades the assignment, the `comment` field will be reset. Then, when they reopen the page, a new room is created and the field updated?

May I know what is the plan for this in the next month? Are we gonna drop `comments` entirely and have a new field? As @geshuming has pointed out, this field is currently ambiguous and it feels like a hacky solution to get the chat out quickly. 

Furthermore, with regards to the workshop, we can deploy a version of a branch also! (it does not have to merged into master necessarily). It is better to do it right than postpone it to a later stage (there is a risk of nobody fixing it and the issue causing further problems down the road).

Let me know your thoughts on this.

Reply:
Shuming pointed this out in our backend PR. I will fix it once I get off work. As for deprecating `comment`, we can have it done over the next week or so. However, it should not be expected to be ready before the workshop.

Reply:
Okay thats fine! Do log an issue for deprecating `comment` either in `cadet` or here for documentation purposes.

We can proceed to move on with this PR.